avaframe/in1Data/computeFromDistributionCfg.ini

```diff
@@ -17,3 +17,5 @@ support = 10000
 name = scenario70
 # if min max value shall be included true, default False
 flagMinMax = False
+# name of distribution
```

Comment:
please list the possible options in the comment ie. [Pert, Uniform...] or whatever

Reply:
Done